django/cantusdb_project/main_app/models/base_chant.py

```
@@ -208,6 +224,50 @@ class Meta:
 
     other_fields_proofread = models.BooleanField(blank=False, null=False, default=False)
 
+    # The value of `manuscript_full_text_std_proofread` as last read from the
+    # database, so the incipit signal can tell when the standardized-spelling
+    # full text is *first* marked proofread and re-sync the incipit only then
+    # (issue #1803). A loaded value is one of `True`/`False`/`None`; a database
+    # `NULL` is a known "not proofread" state. `UNKNOWN_PROOFREAD_STATE` means
+    # the field was never loaded — the instance was built in memory, or the
+    # field was deferred — and the signal must not assume a transition then.
+    _std_proofread_at_load: "bool | None | _UnknownProofreadState" = (
```

Ruff (0.16.1)

[warning] `None` not at the end of the type union.

Move `None` to the end of the type union

(RUF036)